l10n_ch_payment_slip/wizard/isr_batch_print.py

```diff
@@ -1,22 +1,21 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import models, fields, api
 from odoo.exceptions import UserError
 
 
-class BvrBatchPrintWizard(models.TransientModel):
+class ISRBatchPrintWizard(models.TransientModel):
 
-    _name = 'bvr.batch.print.wizard'
+    _name = 'isr.batch.print.wizard'
 
     invoice_ids = fields.Many2many(comodel_name='account.invoice',
                                    string='Invoices')
     error_message = fields.Text('Errors')
```

Comment:
Hi @yvaucher, I think it is a silly mistake but,

When you get the related error in the batch print wizard, the error is editable, in my point of view it is a little bit annoying.

Regards!

Reply:
@BT-ccebrian makes sens to me.

@grindtildeath can you also add a readonly on that field? AFAIK it should only be set server side.

Reply:
Fixed in dee27ae205486dfc0b8e6f217a657567086ed1e3